packages/stargate/src/signingstargateclient.ts

```diff
@@ -102,7 +99,6 @@ function createDefaultTypes(prefix: string): AminoConverters {
     ...createStakingAminoConverters(prefix),
     ...createIbcAminoConverters(),
     ...createFreegrantAminoConverters(),
-    ...createVestingAminoConverters(),
```

Comment:
This should also remain here, such that the new `"not_supported_by_chain"` value is found, which gives a better error message